### ISSUE TITLE
Aciidoctorj-1.6.0 table support does not seem to be implemented yet

### ISSUE THREAD

**Issue by ysb33r** [asciidoctor/asciidoctorj#327]
Given a simple doc

```
= Tables

////
Most of these test cases come straight from the Asciidoctor User Manual
////

== Simple Table

.Simple Table
|===

| Cell in column 1, row 1 | Cell in column 2, row 1

| Cell in column 1, row 2 | Cell in column 2, row 2

| Cell in column 1, row 3 | Cell in column 2, row 3

|===

```
Conversion will fail with exception. The  relevant part of the stacktrace is:

```
java.lang.IllegalArgumentException: Don't know what to do with a #<Asciidoctor::Table:0x4f1fded7>
	at org.asciidoctor.ast.NodeConverter.createASTNode(NodeConverter.java:54)
	at org.asciidoctor.converter.ConverterProxy.convert(ConverterProxy.java:86)
	at org.jruby.internal.runtime.methods.JavaMethod$JavaMethodN.call(JavaMethod.java:669)
	at org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:206)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:168)
	at org.jruby.ast.CallOneArgNode.interpret(CallOneArgNode.java:57)
	at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105)
	at org.jruby.ast.BlockNode.interpret(BlockNode.java:71)
	at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74)
	at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:139)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:306)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:136)
	at org.jruby.ast.CallNoArgNode.interpret(CallNoArgNode.java:60)
	at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105)
	at org.jruby.evaluator.ASTInterpreter.INTERPRET_BLOCK(ASTInterpreter.java:112)
	at org.jruby.runtime.Interpreted19Block.evalBlockBody(Interpreted19Block.java:206)
	at org.jruby.runtime.Interpreted19Block.yield(Interpreted19Block.java:157)
	at org.jruby.runtime.Block.yield(Block.java:142)
	at org.jruby.RubyArray.collect(RubyArray.java:2399)
	at org.jruby.RubyArray.map19(RubyArray.java:2412)
	at org.jruby.runtime.callsite.CachingCallSite.callBlock(CachingCallSite.java:143)
	at org.jruby.runtime.callsite.CachingCallSite.callIter(CachingCallSite.java:154)
	at org.jruby.ast.CallNoArgBlockNode.interpret(CallNoArgBlockNode.java:64)
	at org.jruby.ast.CallOneArgNode.interpret(CallOneArgNode.java:57)
	at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105)
	at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74)
	at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:112)
	at org.jruby.gen.InterfaceImpl104103018.content(org/jruby/gen/InterfaceImpl104103018.gen:13)
	at org.asciidoctor.ast.AbstractBlockImpl.getContent(AbstractBlockImpl.java:74)
```

**Comment by robertpanzer** [comment]
Yes, this is still open.
I remember @mojavelinux having said that the table API is not that final that it would be worth putting an API on top of it.

The next missing classes would be Column, Row and Cell.

**Comment by mojavelinux** [comment]
It may be time. The API isn't perfect, but if we're going to allow converters to be written in Java, we have to bite this bullet. Perhaps we can indicate that the API is alpha and that it's mapped as a prototype.

**Comment by robertpanzer** [comment]
Opened PR #335 to add support for tables.

**Comment by mojavelinux** [comment]
:+1:

**Comment by robertpanzer** [comment]
Fixed with #335
